### DISCUSSION TITLE
Combat, clearly indicate which hexagons will be affected by the area of effect of the Lich shot [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2956] [Ideas]
In fheroes2, it would be nice to display the area of effect of the Lich shot (like the seven hexes of the Fireball spell for example).

With the option "Battle: undead creatures are immune to the area of effect of Lich shot" activated, the hexagon is not grayed out when a troop is immune. (https://github.com/ihhub/fheroes2/discussions/2957)

**Comment by ihhub** [comment]
I agree with this enhancement as not every player even know about this shooter.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
